src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const thumbGallery = document.querySelectorAll('.gallery__thumb');
+
+thumbGallery.forEach(item =>
+  item.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const urlImage = `images${e.target.closest('.list-item__link').href
+      .split('images')[1]}`;
+
```

Comment:
it can be simplified
```suggestion
    const urlImage = e.target.closest('.list-item__link').href;

```